tests/integration/expected_test_results/export/0850032917148.json

```diff
@@ -38,7 +38,7 @@
    "ingredients_text_pt" : "",
    "iron_unit" : "",
    "iron_value" : "",
-   "labels" : "Vegetarian, No preservatives, Vegan",
```

Comment:
if you merge the current version of "main" in your branch, I think it should get rid of those diffs